lib/types/universal.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @fileoverview typings for "eslint/universal" module
+ * @author 唯然<weiran.zsd@outlook.com>
+ */
+
+export type { Linter } from "./index.d.ts";
```

Comment:
To fix the above error:

```suggestion
export { Linter } from "./index";
```